assets/js/modules/analytics/components/common/EnableUniversalAnalytics.js

```diff
@@ -100,6 +106,31 @@ export default function EnableUniversalAnalytics( {
 		revertPropertyAndProfileIDs,
 	] );
 
+	const [ isLookingForMatch, setIsLookingForMatch ] = useState( false );
```

Comment:
Instead of using a local state to control the progress bar, can we use the `modules/analytics` `hasFinishedResolution` selector for the `getProperties` resolver with the current `accountID`? I think that's the only resolver that is used by the `analytics` `findMatchedProperty` action.

I think this is the pattern we mostly use elsewhere.

Reply:
Updated.